order-service/src/main/java/com/innowise/orderservice/model/entity/OrderItem.java

```diff
@@ -0,0 +1,73 @@
+package com.innowise.orderservice.model.entity;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.proxy.HibernateProxy;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString(exclude = {"order", "item"})
+@Table(name = "order_items")
+@Entity
+public class OrderItem extends BaseEntity {
+
+  @Column(name = "quantity", nullable = false)
+  private Integer quantity;
+
+  @ManyToOne(
+      fetch = FetchType.LAZY,
+      cascade = CascadeType.ALL
```

Comment:
Using CascadeType.ALL on ManyToOne relationships with Item and Order is dangerous
This means deleting an OrderItem could cascade delete the Item and Order, which violates business logic